pkg/services/ngalert/migration/alert_rule.go

```diff
@@ -27,8 +28,10 @@ type alertRule struct {
 	Data            []alertQuery
 	IntervalSeconds int64
 	Version         int64
-	UID             string `xorm:"uid"`
-	NamespaceUID    string `xorm:"namespace_uid"`
+	UID             string  `xorm:"uid"`
+	NamespaceUID    string  `xorm:"namespace_uid"`
+	DashboardUID    *string `xorm:"dashboard_uid"`
```

Comment:
Added because `AddDashboardUIDPanelIDMigration` used to be conditional on migration (runs and reverts with migration). Since we cna't rely on it anymore the migration service needs to populate the fields. See 2fdf1d06e25939ad9ef623f21fe37b96078420ed